tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_jump_opcodes.py

```diff
@@ -0,0 +1,166 @@
+            None,
+            id="jump_tables_valid",
+        ),
+        pytest.param(
+            # Check that jumps into the middle on an opcode are not allowed
+            Container(
+                name="EOF1I0019",
+                sections=[
+                    Section.Code(
+                        code=Op.PUSH1(1)
+                        + Op.RJUMPV[b"\x02\x00\x02\xFF\xFF"]
+                        + Op.ADDRESS
+                        + Op.POP
+                        + Op.ADDRESS
+                        + Op.POP
+                        + Op.STOP,
+                    ),
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef0001010004020001000D04000400008000016001E2020002FFFF30503050000bad60A7",
+            EOFException.INVALID_RJUMP_DESTINATION,
+            id="rjump_invalid",
+        ),
+        pytest.param(
+            # TODO why here is expected an exception by the comment but test is valid
+            # Check that you can't get to the same opcode with two different stack heights
+            Container(
+                name="EOF1I0020",
+                sections=[
+                    Section.Code(
+                        code=Op.PUSH1(1) + Op.RJUMPI[1] + Op.ADDRESS + Op.NOOP + Op.STOP,
+                    ),
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef0001010004020001000804000400008000016001E10001305B000bad60A7",
+            None,
+            id="jump_to_opcode_ok",
+        ),
+        pytest.param(
+            # Check that jumps into the middle on an opcode are not allowed
+            Container(
+                name="EOF1I0019",
+                sections=[
+                    Section.Code(code=Op.RJUMP[3] + Op.RJUMP[2] + Op.RJUMP[-6] + Op.STOP),
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef0001010004020001000A0400040000800000E00003E00002E0FFFA000bad60A7",
+            EOFException.INVALID_RJUMP_DESTINATION,
+            id="rjump_3_2_m6_fails",
+        ),
+        pytest.param(
+            # Check that jumps into the middle on an opcode are not allowed
+            Container(
+                name="EOF1I0019",
+                sections=[
+                    Section.Code(
+                        code=Op.PUSH1(0)
+                        + Op.PUSH1(0)
+                        + Op.PUSH1(0)
+                        + Op.RJUMPI[3]
+                        + Op.RJUMPI[2]
+                        + Op.RJUMPI[-6]
+                        + Op.STOP,
+                    ),
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef000101000402000100100400040000800003600060006000E10003E10002E1FFFA000bad60A7",
+            EOFException.INVALID_RJUMP_DESTINATION,
+            id="push1_0_0_0_rjump_3_2_m6_fails",
+        ),
+    ],
+)
+def test_jump_opcodes(
```

Comment:
Once case I could not find that should be added to `tests/prague/eip7692_eof_v1/eip4200_relative_jumps/test_rjumpi.py` is from https://github.com/ipsilon/eof/blob/main/spec/eof.md#stack-validation 4.i:
> This implies that the last instruction may be a terminating instruction or RJUMP

We should add a check that `RJUMPI` nor `RJUMPV` cannot be at the end of the bytecode (we do have tests that place `RJUMP` at the end of the bytecode).
E.g.
```python
Op.PUSH1(0) + Op.PUSH1(0) + Op.RJUMPI[1] + Op.STOP + Op.RJUMPI[-3]
```
should be invalid.